v3/src/components/data-display/data-display-types.ts

```diff
@@ -6,6 +6,7 @@ export type Point = { x: number, y: number }
 export type CPLine = { slope: number, intercept: number, pivot1?: Point, pivot2?: Point }
 export const kNullPoint = {x: -999, y: -999}
 
+export type BinOption = "binWidth" | "binAlignment"
```

Comment:
I would leave this definition in `display-config-palette.tsx` since that's the only place it's used. It's not a constant meant to be shared across modules.